uber/site_sections/emails.py

```diff
@@ -16,18 +16,28 @@ def sent(self, session, **params):
     sent.restricted = [c.PEOPLE, c.REG_AT_CON]
 
     def pending(self, session, message=''):
-        approved = {ae.ident for ae in session.query(ApprovedEmail).all()}
+        automated_emails = []
+        for automated_email in AutomatedEmail.instances.values():
+            automated_emails.append({
+                'automated_email': automated_email,
+                'num_sent': session.query(Email).filter_by(ident=automated_email.ident).count(),
+                'last_run_results':
+                    SendAllAutomatedEmailsJob.last_result[automated_email.ident]
+                    if SendAllAutomatedEmailsJob.last_result else None
```

Comment:
Shorter version:

```
SendAllAutomatedEmailsJob.last_result.get(automated_email.ident, None)
```

since ``last_result`` is initialized to an empty dict.